mne/surface.py

```diff
@@ -1410,3 +1411,50 @@ def _complete_sphere_surf(sphere, idx, level, complete=True):
         complete_surface_info(surf, copy=False)
     surf['coord_frame'] = sphere['coord_frame']
     return surf
+
+
+@verbose
+def check_coreg(info, trans, subject, subjects_dir=None, dig_kinds='auto',
+                verbose=None):
+    """Compute distances between head shape points and the scalp surface.
+
+    This function is useful to check that coregistration is correct.
+    Unless outliers are present in the head shape points,
+    one can assume an average distance around 2-3 mm.
+
+    Parameters
+    ----------
+    info : instance of Info
+        The measurement info that contains the head shape
+        points in ``info['dig']``.
+    trans : str | instance of Transform
+        The head<->MRI transform. If str is passed it is the
+        path to file on disk.
+    subject : str
+        The name of the subject.
+    subjects_dir : str | None
+        Directory containing subjects data. If None use
+        the Freesurfer SUBJECTS_DIR environment variable.
+    %(dig_kinds)s
+    %(verbose)s
+
+    Returns
+    -------
+    dists : array, shape (n_points,)
+        The distances.
+
+    See Also
+    --------
+    get_fitting_dig
+
+    Notes
+    -----
+    .. versionadded:: 0.19
+    """
+    from .bem import get_fitting_dig
+    pts = get_head_surf(subject, 'head', subjects_dir=subjects_dir)['rr']
+    trans = _get_trans(trans, fro="mri", to="head")[0]
+    pts = apply_trans(trans, pts)
+    info_dig = get_fitting_dig(info, dig_kinds, exclude_frontal=False)
+    dists = _compute_nearest(pts, info_dig, return_dists=True)[1]
+    return dists
```

Comment:
Simplified code by using existing functions. Also moved to `surface.py` instead of `transforms.py` because more similar functionality exists in `surface.py` than `transforms.py` (e.g., knowledge of subjects, how transformations should be combined, etc.).